mscore/dragelement.cpp

```diff
@@ -85,8 +85,10 @@ void ScoreView::doDragElement(QMouseEvent* ev)
 void ScoreView::endDrag()
       {
       for (Element* e : _score->selection().elements()) {
-            e->endDrag(editData);
-            e->triggerLayout();
+            if (e == editData.element) { // make sure element wasn't deleted before drag is ended
```

Comment:
Wouldn't this break dragging multiple elements?

Reply:
So dragging multiple elements is possible? How?

Reply:
Select one element or several elements, press Ctrl, start dragging another (yet unselected) element. After that you can release Ctrl button, several elements should be dragged. Not that it works perfectly but still works, and maybe in the future this would become more usable.

Reply:
I get it now. So I guess I just have to reverse this while keep the change in `element.cpp`.